docs/manual/introduction.tex

```diff
@@ -714,6 +714,8 @@
   These annotations may reduce the number of type-checking
   errors when running type-checking in the future; see
   Section~\ref{whole-program-inference}.
+  Using \<-Ainfer> produces \<.jaif> files.
```

Comment:
How about deprecating usage of `-Ainfer` in favor of `-Ainfer=jaif`?

Reply:
I don't like the idea of breaking backwards-compatibility for no particular benefit. `-Ainfer=jaif` does work with this change; I'm happy to change the manual to suggest using that version.